keras_cv/models/generative/stable_diffusion/stable_diffusion.py

```diff
@@ -281,6 +300,14 @@ def _get_unconditional_context(self):
 
         return unconditional_context
 
+    @property
+    def image_encoder(self):
```

Comment:
I think if we're adding an image_encoder property, we may also want an API surface to use it (e.g. `encode_image`)?

Reply:
I was thinking that you just run `stable_diffusion.image_encoder.predict()`.  

Perhaps this is not a great API surface though?

Reply:
Seems fine!